integrationtest/spring/pom.xml

```diff
@@ -0,0 +1,72 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.hibernate.search</groupId>
+        <artifactId>hibernate-search-integrationtest</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+    <artifactId>hibernate-search-integrationtest-spring-repackaged</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Hibernate Search ITs - Spring Repackaged JAR</name>
+    <description>Testing if Hibernate Search will start correctly inside a Spring's repackaged jar</description>
+
+    <modules>
+        <module>model</module>
+        <module>application</module>
+        <module>test</module>
+    </modules>
+
+    <properties>
+        <test.elasticsearch.run.skip>${test.elasticsearch.connection.uris.defined}</test.elasticsearch.run.skip>
+        <test.database.run.skip>false</test.database.run.skip>
```

Comment:
This is supposed to enable starting DB containers using docker-maven-plugin. But from what I can see you hardcoded H2 as the database for this project, so you don't need this.

IMO you should either adapt the project configuration to work with non-H2 databases (probably not worth it), or use `true` here with a comment explaining that we just test H2.

Reply:
ohh good point! I thought I've removed these. Yeah no need to have this for various backends or DBs as the test isn't really dependant on them.